src/middleware.js

```diff
@@ -61,20 +67,28 @@ function init(swaggerPath, options) {
                     }
                 });
         });
-    })
-        .catch(function (error) {
-            return Promise.reject(error);
-        });
 }
+
+function init(swaggerPath, options) {
```

Comment:
we should add comment that this is kept for backward compatibility and it's not really async